runtime/src/snapshot_bank_utils.rs

```diff
@@ -61,7 +64,7 @@ pub fn bank_fields_from_snapshot_archives(
     full_snapshot_archives_dir: impl AsRef<Path>,
     incremental_snapshot_archives_dir: impl AsRef<Path>,
     accounts_db_config: &AccountsDbConfig,
-) -> snapshot_utils::Result<BankFieldsToDeserialize> {
+) -> SnapshotResult<BankFieldsToDeserialize> {
```

Comment:
Same here: I think we should go back to `agave_snapshots::Result`.

Reply:
done